README.md

```diff
@@ -32,16 +33,27 @@ Eventually there'll be one of the following:
 There is currently no parsable trace-file output, and replaying traces is not possible.
 
 
-### Usage
-
-This project uses [xboxpy](https://github.com/XboxDev/xboxpy).
-Please read its documentation to find out how to install and configure it for your Xbox.
-
-Afterwards, you can run these commands:
-
+### Building
+
+1. This project uses [xboxpy](https://github.com/XboxDev/xboxpy).
```

Comment:
Should probably include a note that this only works with XBDM (although it might not work with the others in its current state anyway)